yeat/cli/illumina.py

```diff
@@ -10,6 +10,18 @@
 from argparse import ArgumentTypeError
 
 
+def fastp_options(parser):
+    illumina = parser.add_argument_group("fastp arguments")
+    illumina.add_argument(
+        "-l",
+        "--length-required",
+        type=int,
+        metavar="L",
+        default=50,
+        help="discard reads shorter than the required L length during fastp; by default L=50",
+    )
```

Comment:
Users can now adjust the `-l` or `--length-required` during the `fastp` step through the command line flags.

Reply:
I'd suggest replacing "during fastp" with "after pre-processing" or something like that.